web/packages/teleterm/src/ui/AccessRequestCheckout/AccessRequestCheckout.tsx

```diff
@@ -170,6 +170,7 @@ export function AccessRequestCheckout() {
                       switch (c.kind) {
                         case 'app':
                         case 'saml_idp_service_provider':
+                        case 'aws_ic_account_assignment':
```

Comment:
Is there a way to visually distinguish AWS Identity Center Teleport Applications (Account → Permission Set apps) from regular Teleport Apps in the UI?

In the current design, it’s hard for users to understand the different behaviors for different app types:
![Screenshot 2024-12-06 at 11 25 06](https://github.com/user-attachments/assets/b93e4f38-420d-4a2a-aada-71fb972be488)

cc @roraback

Reply:
Good call. The UI currently tries to guess icons based on the name. See the IC account that has `aws` word in it shows the aws icon. 
<img width="1783" alt="Screenshot 2024-12-06 at 9 40 08 AM" src="https://github.com/user-attachments/assets/85ae39b8-c636-47a2-857d-0f3c5e285ce2">

I was wondering if the `teleprt.dev/origin:aws-identity-center` label would be helpful to distinguish but I can see it may not be much helpful. I vaguely remember at one point that the UI also tried to guess icons from metadata icon label. Let me check if that can be extended to read from our origin label.

Reply:
I am keeping this aside for now, linking the other AWS icon ticket to follow up later https://github.com/gravitational/teleport.e/issues/5467

Reply:
Icon added in this PR https://github.com/gravitational/teleport/pull/50256